xpdacq/Ni24.D

```diff
@@ -0,0 +1,27 @@
+# Calibrant: Nickel (Ni)
```

Comment:
@sbanerjee23  could you do this (submit as a patch to pyFAI)? Take entire credit :)